src/man/sssd.conf.5.xml

```diff
@@ -1446,15 +1446,16 @@ pam_account_locked_message = Account locked, please contact help desk.
                     <term>pam_cert_db_path (string)</term>
                     <listitem>
                         <para>
-                            The path to the certificate database which contain
-                            the PKCS#11 modules to access the Smartcard.
+                            The path to the certificate database.
                         </para>
                         <para>
                             Default:
                             <itemizedlist>
                                 <listitem><para>/etc/pki/nssdb (NSS version,
-                                                path to a NSS
-                                                database)</para></listitem>
+                                                path to a NSS database which
+                                                contains the PKCS#11 modules
+                                                to access the Smartcard)
```

Comment:
Hi,

please add '... to access the Smartcard `and the trusted CA certificates`'

bye,
Sumit

Reply:
Done.